packages/Webkul/Shop/src/Resources/views/emails/layouts/logo.blade.php

```diff
@@ -0,0 +1,5 @@
+@if (core()->getConfigData('general.design.admin_logo.logo_image'))
+    <img src="{{ \Illuminate\Support\Facades\Storage::url(core()->getConfigData('general.design.admin_logo.logo_image')) }}" alt="Bagisto" style="height: 40px; width: 110px;"/>
```

Comment:
@rahulshukla-webkul Can use this only: \Storage::url(core()->getConfigData('general.design.admin_logo.logo_image'))